.gitignore

```diff
@@ -133,3 +133,5 @@ dmypy.json
  
 # DS_Store 
 .DS_Store
+
+db.sqlite3
```

Comment:
should add `.idea`

Reply:
and please remove all the .idea files from this PR like:

- .idea/.gitignore
- .idea/commune.iml
- .idea/inspectionProfiles/profiles_settings.xml
- ...